pyVHDLModel/__init__.py

```diff
@@ -41,4 +41,4 @@
 :copyright: Copyright 2007-2021 Patrick Lehmann - Bötzingen, Germany
 :license: Apache License, Version 2.0
 """
-__version__ = "0.7.7"
+__version__ = "0.8.0"
```

Comment:
Maybe `doc/conf.py` and `setup.py` can read the version from here?

Reply:
This will come in a future iteration.